src/main/java/com/google/devtools/build/lib/analysis/actions/StrippingPathMapper.java

```diff
@@ -174,7 +185,7 @@ public ArgChunk mapCustomStarlarkArgs(ArgChunk chunk) {
 
     // TODO: b/327187486 - This materializes strings when totalArgLength() is called. Can it
```

Comment:
@aranguyen Can we get rid of these special cased actions or are they still needed?

Reply:
We'll keep these for now.